gce/notebook/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM daskdev/dask-notebook
+    
+RUN conda install --yes -c conda-forge \
+    xarray \
+    netcdf4 \
+    scipy \
+    && conda clean -tipsy
```

Comment:
Maybe we should be using a standard environment file here?

Reply:
Sure, I would be fine with that